### DISCUSSION TITLE
area/metrics

### DISCUSSION THREAD

**Discussion by Kameshwarsingh** [quarkusio/quarkus#32987] [Q&A] [answered]
area/metrics 

Is there any performance data/url/previous-discussions indicating resource utiltiations
    - How much CPU, memory is taken by metrics collection and transportation?
    Example - Ratio of CPU utlization by real-business-work (resource) VS  CPU utilization by metrics collection and transportation?
This will help to understand CPU, memory utilization by various parts of components.

[a comment by an automated account is not shown]

**Comment by ebullient** [comment]
A starting place is here (only covering micrometer): https://micrometer.io/docs/concepts#_memory_footprint_estimation

We have additional performance/resource utilization measurements for telemetry across the board on the list of things to do. The thing to note: telemetry of any kind isn't free. There is always a balance.
